admin/backend/routes.py

```
@@ -0,0 +1,768 @@
+            cursor = conn.cursor()
+            cursor.execute("SELECT 1")
+            cursor.fetchone()
+
+        return {"status": "healthy", "timestamp": datetime.now().isoformat()}
+    except Exception as e:
+        raise HTTPException(status_code=503, detail=f"Database connection failed: {str(e)}")
+
+
+# Knowledge base management endpoints
+@router.post("/knowledge/upload")
+async def upload_knowledge_files(files: List[UploadFile] = File(...), _: None = Depends(verify_admin_token)):
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)